sql/core/src/test/scala/org/apache/spark/sql/CTEInlineSuite.scala

```diff
@@ -735,6 +736,86 @@ abstract class CTEInlineSuiteBase
       checkAnswer(df, Row(1))
     }
   }
+
```

Comment:
shall we update the test name of the origin `SPARK-49816` test?

Reply:
Fixed in: https://github.com/apache/spark/pull/49379/commits/c62c6c4bf2ec4591fee1c5129315a2d9878159a9